src/main/kotlin/org/openrewrite/kotlin/KotlinTypeSignatureBuilder.kt

```diff
@@ -345,11 +345,6 @@ class KotlinTypeSignatureBuilder(private val firSession: FirSession) : JavaTypeS
             s.append("Generic{in ")
             s.append(signature(type1))
             s.append("}")
-        } else if (type is ConeCapturedType) {
```

Comment:
Hi @knutwannheden, I'd like an extra pair of eyes here if I overlook why this was added.
AFAICT, the signature(type1) on line 351 always returns `Generic{in {undefined}}` and the ConeCaptureType was covered on line `392`.